rice-middleware/kns/src/main/java/org/kuali/rice/kns/web/struts/action/KualiExceptionHandlerAction.java

```diff
@@ -176,9 +198,38 @@ protected ActionForward processException(ActionMapping mapping,
 		// Get exception properties from the Http Request
 		Map<String, String> properties = (Map<String, String>) request
 				.getAttribute(IncidentReportUtils.EXCEPTION_PROPERTIES);
-		// Construct the exception incident object
+
+        // Construct the exception incident object
 		KualiExceptionIncident ei = reporterService.getExceptionIncident(
 				exception, properties);
+
+        // For Production, hide sensitive data in user session
+        if(!ConfigContext.getCurrentContextConfig().getDevMode()) {

```

Comment:
Probably should be using ConfigContext.getCurrentConfig.isProductionEnvironment()